worker-languagedetection/README.md

```diff
@@ -41,6 +59,13 @@ e.g. DetectedLanguage1\_ConfidencePercentage:100
 
 - DetectedLanguages_ReliableResult : A boolean flag that signals whether the result is reliable. 
 
+#### Multi-Field Processing:
+
+- `WORKER_LANG_DETECT_FIELD_PREFIX$FieldName`: 
+   This field will contain two values at present, the first value in this string will be the percentage of the detected language, e.g. `98%`. The second value in the string after a space will be the language that was detected language code, e.g. `en`. If a field is multi value this result will be a combined result for all of the values.
```

Comment:
This really needs an example output field name and an example output value showing the percentage with the language code.